app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -698,6 +698,67 @@ ControllerContainer::SetFrameId(const uint64_t aFrameId) {
   m.immersiveFrameId = aFrameId;
 }
 
+// Helper function (similar to the one in DeviceDelegateOpenXR.cpp)
+static vrb::Matrix XrPoseToMatrix(const XrPosef& pose) {
```

Comment:
ControllerContainer is backend-agnostic there can not be any reference to OpenXR here